externs/es6.js

```diff
@@ -447,12 +448,13 @@ var BufferSource;
 
 
 /**
- * @constructor
+ * @interface
  * @implements {IArrayLike<number>}
  * @implements {Iterable<number>}
- * @extends {ArrayBufferView}
+ * @implements {ArrayBufferView}
```

Comment:
Sorry, I thought I saw in a related type that interface should be used in @implements tags (contrary to how Java would express this). I'm not seeing that now, so I was probably mistaken.

This was also meant to be something of a draft, looking to see if this change would be acceptable - as I suggested in the description, it might be more correct to use `@typedef`.